portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/searchportlet/Search.testcase

```diff
@@ -385,6 +385,47 @@ definition {
 			searchTerm = "");
 	}
 
+	@description = "This is a use case for LPS-179990"
+	@priority = 3
+	test CanRemoveSortFieldsAndDeactivateRelevanceInSortWidget {
+		task ("Open search page and add a new Sort Widget") {
+			Search.openSearchPage();
+
+			SearchPortlets.addWidgetsViaUI(searchPortletList = "Sort");
+
+			Refresh();
+		}
+
+		task ("Open Sort Widget configuration") {
+			Portlet.gotoPortletOptions(
+				portletName = "Sort",
+				portletOption = "Configuration");
+
+			SelectFrame(locator1 = "IFrame#CONFIGURATION");
+		}
+
+		task ("Delete all indexed field names and disable relevance") {
+			var count = 0;
+
+			while (${count} != 6) {
+				Click(locator1 = "Button#TRASH_ENABLED");
+
+				var count = ${count} + 1;
+			}
```

Comment:
This is not reliable if there are new indexed fields. The while loop should continue clicking `Button#TRASH_ENABLED` if it still exists.